mlflow/utils/databricks_utils.py

```diff
@@ -630,6 +630,19 @@ def get_databricks_workspace_info_from_uri(tracking_uri: str) -> Optional[Databr
         return None
 
 
+def check_databricks_secret_scope_access(scope_name):
+    dbutils = _get_dbutils()
+    if dbutils:
+        try:
+            dbutils.secrets.list(scope_name)
+        except Exception as e:
```

Comment:
nit: Is there a way to have more specific check here? That triggers only on access denied errors?

Reply:
Hmm based on the rest of the file it seems we only check for a generic `Exception`, which is also what I see for most of MLflow. We can check if the string contains "Permission Denied" I guess but this might break with convention and is also brittle itself. Thoughts @harupy ?

Reply:
@arpitjasa-db what kinds of exception(s) does `dbutils.secrets.list()` throw? This isn't an MLflow consideration (i.e. we don't care how MLflow handles its own exceptions), since the code you're try / catching comes from dbutils.

Reply:
The error would look something like
```
com.databricks.common.client.DatabricksServiceHttpClientException: PERMISSION_DENIED: User <user> does not have READ permission on scope <scope> to perform this action.
```

Reply:
Got it. If we make this a warning and print the full exception contents, then catching exception SGTM

Reply:
If we replace with a warning should we return false so this check still fails in the boolean expression where it's called? Just worried the warning might not be noticed, especially if this isn't run manually

Reply:
I don't think so. I think we should still write the secrets yaml.

I'm thinking about the use case where a model is trained in one workspace and deployed in another. In that case, it would be entirely reasonable for the secret not to be defined in the training workspace.

Reply:
Yeah that's a good callout, I'll update to non-blocking warning